src/styles/header.scss

```
@@ -0,0 +1,60 @@
+.logo {
+  @include square($logoSize);
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    display: inline-block;
+    margin-left: 20px;
+
+    &:first-child {
+      height: 60px;
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    display: inline;
```

displa
```suggestion
    display: block;
```
it will help to set vertical paddings